### ISSUE TITLE
Rename not updating dependencies

### ISSUE THREAD

**Issue by ChuckSavage** [modesttree/Projeny#56]
I renamed a Package (from within the Projeny Package Manager), but the a package that depended on it, didn't get updated. So when I do Update Directories it complains that it can't find the OldNamed package anywhere. 

Or is this something I have to do manually? Seems odd that we have to create the file manually to begin with.

**Comment by svermeulen** [comment]
Yes, currently packages are identified entirely by their name, so if you rename the folder then you have to update all the config files that are pointing to it in ProjenyProject.yaml or ProjenyPackage.yaml files.

Yeah, there is no GUI for the ProjenyPackage.yaml file - that would be nice to have eventually.  If we could migrate entirely to using the GUI for managing dependencies then that might also allow us to store an ID rather than a name in the ProjenyPackage.yaml file and then support renames
